crates/bin/pcli/src/command/view/tx.rs

```diff
@@ -382,8 +382,19 @@ impl TxCmd {
                         "Claim Liquidity Position Reward".to_string(),
                         "".to_string(),
                     ],
-                    penumbra_transaction::ActionView::Ics20Withdrawal(_) => {
-                        ["Ics20 Withdrawal".to_string(), "".to_string()]
+                    penumbra_transaction::ActionView::Ics20Withdrawal(w) => {
+                        let unit = w.denom.best_unit_for(w.amount);
```

Comment:
Side question: the `Ics20Withdrawal` domain type has an `asset::DenomMetadata`, but the ICS20 packet only has a `string denom`, correct? How does the domain type get the asset metadata? @aubrika